Azure-Samples/azure-search-openai-demo
# Locust file related improvements

## Purpose

This PR makes a few improvements around the locustfile for loadtesting:

* Fixes the test data to use the new REST Chat App protocol
* Moves productionizing section into its own dedicated document, as we've gotten feedback that the README is too long, and this enables us to go into more details.
* Adds a loadtesting section to productionizing document.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Click from README to productionizing guide
* See image load correctly
* Click from productionizing guide to auth section
